integration_test/gallery_compile_test.dart

```diff
@@ -8,8 +8,8 @@ import 'package:test/test.dart';
 import 'package:path/path.dart' as path;
 
 // Benchmark size in kB.
-const int bundleSizeBenchmark = 4100;
-const int gzipBundleSizeBenchmark = 1004;
+const int bundleSizeBenchmark = 4124;
+const int gzipBundleSizeBenchmark = 1008;
```

Comment:
I think it's fine to increase this a bit more, so we're do not have to update it all the time. Perhaps to 4500 and 1200?

Reply:
sounds good to me. I'll update it.